rust_src/src/editfns.rs

```diff
@@ -420,7 +420,7 @@ pub fn propertize(args: &[LispObject]) -> LispObject {
 
     // the unwrap call is safe, the number of args has already been checked
     let first = it.next().unwrap();
-    let orig_string = first.as_string_or_error();
+    let orig_string = LispStringRef::from(*first);
```

Comment:
Why use `From` here and `Into` in the previous change?

Reply:
I tried it a few different ways and this is the only one that worked. Something about the dereference I guess.